src/components/RightSidebar/Participants/ParticipantsList/Participant/Participant.vue

```diff
@@ -144,6 +145,16 @@ export default {
 			// source: groups, circles
 			return 'icon-contacts'
 		},
+		callIconClass() {
+			if (this.participant.inCall === PARTICIPANT.CALL_FLAG.DISCONNECTED) {
+				return ''
+			}
+			const hasVideo = this.participant.inCall & PARTICIPANT.CALL_FLAG.WITH_VIDEO
```

Comment:
```suggestion
			const hasVideo = this.participant.inCall && PARTICIPANT.CALL_FLAG.WITH_VIDEO
```
That was always evaluating true
![Screenshot_20200123_130511](https://user-images.githubusercontent.com/26852655/72983259-253e5780-3de1-11ea-8df6-96bd607391a0.png)

Reply:
No, we want to check if the binary is set.